types/polka/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Piotr Kuczynski <https://github.com/pkuczynski>
 //                 James Messinger <https://github.com/JamesMessinger>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// Minimum TypeScript Version: 3.1
+// Minimum TypeScript Version: 3.7
```

Comment:
Why would you bump minimym TS version?

Reply:
Hi @pkuczynski, thanks for the great work you've done with these types.
I got errors when running the tests locally without bumping the version to 3.7 
<img width="1137" alt="Screenshot 2021-01-06 at 21 13 29" src="https://user-images.githubusercontent.com/39765499/103820480-5b153400-5064-11eb-9c51-aa68ca467f92.png">


Same error also appeared in [CI test run](https://github.com/DefinitelyTyped/DefinitelyTyped/runs/1658058249#step:5:97)

Reply:
Makes sense! 👍